site/specs/numbers.json

```diff
@@ -3996,7 +3996,7 @@
04:00\n                 </p>\n             </td>\n             <td valign=\"top\">\n                 <p>\n                     Now\n                 </p>\n             </td>\n             <td valign=\"top\">\n                 <p>\n                     This represents the position in the data set from which you want your output to begin.  If this parameter is omitted, results will include Emergency Notification Recipients last modified on or before the current date and time.  If a date and time is provided, results will include Emergency Notification Recipients last modified on or before that date and time.  Date/time format is YYYY-MM-DDTHH:MM:SS.mmmZ, where the Z indicate UTC time, or YYYY-MM-DDTHH:MM:SS.mmm-hh:mm, where \"-hh:mm\" represents the local time offset, plus or minus, from UTC time.\n                 </p>\n             </td>\n         </tr>\n         <tr>\n             <td  valign=\"top\">\n                 <p>\n                     Size\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                     Size=20\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                     100\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                     The maximum number of Emergency Notification Recipient results to include in the response payload.  This value may be between 1 and 500.\n                 </p>\n             </td>\n         </tr>\n         <tr>\n             <td  valign=\"top\">\n                 <p>\n                   GroupCount\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                    GroupCount=true\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                   false\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                   If GroupCount is true, each notification recipient in the response will include a GroupCount representing how many notification groups it is associated with.\n                 </p>\n             </td>\n         </tr>\n         <tr>\n             <td  valign=\"top\">\n                 <p>\n                   UriSubstring\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                    UriSubstring=foo\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                   No filter\n                 </p>\n             </td>\n             <td  valign=\"top\">\n                 <p>\n                   Returns all Emergency Notification Recipients that contain this value, whether that value is a substring or partial value of the recipient or a recipient TYPE (EMAIL, SMS, TTS, CALLBACK).\n                 </p>\n             </td>\n         </tr>\n </table></p>\n",
```

Comment:
It seems like we're documenting the query parameters in the description instead of listing them in the parameters array.  Can we move them into the parameter array?

Reply:
hey @mrmeyers99 thanks for the feedback! I ran this locally and saw them in the table along with the other query params. I added it in the "description" because the only param in the Parameter array was accountId. What exactly does that parameter array do, and should all of the params that are currently in the description be there? We have only added the EngName and the EnrRecipient there, but saw the others come from the old iris api docs

Reply:
oops, misread which PR this was. The only param added here was the UriSubstring

Reply:
It looks like this whole endpoint is wrong.  The parameters should not be documented in the description, they should be documented in the parameters array.  See `/accounts/{accountId}/externalTns` as an example.  We could probably merge this anyway and that fix the whole operation at once though.

Reply:
Let's go ahead and merge this, I added a todo in the migration to move these parameter tables into the parameter array.